build.gradle

```diff
@@ -13,14 +13,18 @@ buildscript {
 
 //add dependency repositories
 repositories {
-	maven {
-		name = "CB Repo"
-		url = "http://chickenbones.net/maven"
-	}
-	maven {
-		name = "JEI Maven"
-		url = "http://dvs1.progwml6.com/files/maven"
-	}
+    maven {
+        name = "CB Repo"
+        url = "http://chickenbones.net/maven"
+    }
+    maven {
+        name = "JEI Maven"
+        url = "http://dvs1.progwml6.com/files/maven"
+    }
+    maven {
+        name = "CurseForge"
+        url = "https://minecraft.curseforge.com/api/maven/"
+    }
```

Comment:
I understand that and am well aware of how gradle works, this is just more of a headache when setting up my environment, the buildscripts are only used by jenkins and contributors, having the relevant sources in /api/java is just easier as i don't use this script to setup my env, as it stands now i would have to manually download that jar and link it in idea instead of just adding /src/api/java as a source directory. And if you are wondering why i don't use this to setup my env is because im setup for base editing, so i have modules in idea that all revolve around a forge dev workspace, so i can edit basically everything on the fly without needing to do other bs.